recipes/tinycthreadpool/all/conanfile.py

```diff
@@ -0,0 +1,79 @@
+import os
+
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.microsoft import is_msvc_static_runtime, is_msvc
+from conan.tools.scm import Version
+from conan.tools.files import get, copy
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+
+required_conan_version = ">=1.53.0"
```

Comment:
improved toolchain to skip a policy

```suggestion
required_conan_version = ">=1.54.0"
```